### DISCUSSION TITLE
Klick'r 3.0.0

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#468] [Announcements]
Klick'r 3.0.0 is now stable !
Thank you to all testers that have helped greatly and endure the first beta crashes!

All changes contained in this version compered to 2.4.2 can be found in the beta release notes:
* [Beta01](https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta01)
* [Beta02](https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta02)
* [Beta03](https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta03)
* [Beta04](https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta04)
* [Beta05](https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta05)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0'>Klick'r 3.0.0</a>.</em>

**Comment by noobiessss** [comment]
Anything new or fix from beta05 to Stable? Or is it just a name change.
